client/app/telecom/telephony/line/calls/displayNumber/telecom-telephony-line-calls-displayNumber.html

```diff
@@ -84,19 +82,21 @@ <h1 data-translate="loading"></h1>
         <div class="form-group mt-4">
             <div class="row">
                 <div class="col-md-6">
-                    <button
-                        type="reset"
-                        data-translate="cancel"
-                        class="btn btn-sm btn-cancel btn-default"
-                        data-ng-click="LineDisplayNumberCtrl.reset()"
-                        data-ng-disabled="!LineDisplayNumberCtrl.hasChanges() || LineDisplayNumberCtrl.isUpdating">
+                    <hr />
+                    <button type="reset"
+                            data-translate="cancel"
+                            class="btn btn-default"
+                            data-ng-click="LineDisplayNumberCtrl.reset()"
+                            data-ng-disabled="!LineDisplayNumberCtrl.hasChanges() || LineDisplayNumberCtrl.isUpdating">
                     </button>
                     <button type="submit"
-                            class="btn btn-sm btn-primary float-right"
+                            class="btn btn-primary float-right"
                             data-ng-disabled="!displayNumberForm.$valid || !LineDisplayNumberCtrl.hasChanges() || LineDisplayNumberCtrl.isUpdating">
                         <span data-translate="telephony_line_actions_line_calls_submit_changes"></span>
-                        <spinner class="ml-2"
-                                 data-ng-if="LineDisplayNumberCtrl.isUpdating"></spinner>
+                        <bouncing-box-loader class="d-inline-block align-middle mr-2"
```

Comment:
`<bouncing-box-loader>` can be the first child of your `<button>` element. (ref: http://master.ui-kit.ovh/#!/ovh-ui-kit/loader#button)